src/app/src/components/FacilityListItemsTable.jsx

```diff
@@ -48,199 +56,313 @@ const facilityListItemsTableStyles = Object.freeze({
-            {paginationControlsRow}
-            <div style={facilityListItemsTableStyles.tableWrapperStyles}>
-                <Table>
-                    <TableHead>
-                        <FacilityListItemsTableRow
-                            rowIndex="CSV Row Index"
-                            countryName="Country Name"
-                            name="Name"
-                            address="Address"
-                            status="Status"
-                            hover={false}
+        return (
+            <Paper style={facilityListItemsTableStyles.containerStyles}>
+                <div style={facilityListItemsTableStyles.statusFilterStyles}>
+                    <div style={facilityListItemsTableStyles.statusFilterSelectStyles}>
+                        <ReactSelect
+                            isMulti
+                            id="listItemStatus"
+                            name="listItemStatus"
+                            classNamePrefix="select"
+                            options={facilityListStatusFilterChoices}
+                            placeholder="Filter by item status..."
+                            value={createSelectedStatusChoicesFromParams(params)}
+                            onChange={handleChangeStatusFilter}
                         />
-                    </TableHead>
-                    <TableBody>
-                        {tableRows}
-                    </TableBody>
-                </Table>
-            </div>
-            {paginationControlsRow}
-        </Paper>
-    );
+                    </div>
+                    <ShowOnly when={!!(params && params.status)}>
+                        <span style={facilityListItemsTableStyles.statusFilterMessageStyles}>
+                            Showing {filteredCount} of {list.item_count} items.
+                        </span>
+                        <Button color="primary" onClick={handleShowAllClicked}>
+                            Show all items
+                        </Button>
+                    </ShowOnly>
+                </div>
+                {paginationControlsRow}
+                <div style={facilityListItemsTableStyles.tableWrapperStyles}>
+                    <Table>
+                        <TableHead>
+                            <FacilityListItemsTableRow
+                                rowIndex="CSV Row Index"
+                                countryName="Country Name"
+                                name="Name"
+                                address="Address"
+                                status="Status"
+                                hover={false}
+                            />
+                        </TableHead>
+                        <TableBody>
+                            {tableRows}
+                            <ShowOnly when={listIsEmpty}>
+                                <TableRow>
+                                    <TableCell colSpan={5} style={{textAlign: 'center'}}>
```

Comment:
Looks like civicci's seeing a linter error here resolvable by adding a space before and after `textAlign: 'center'`.

Reply:
Addressed in fixup 0aeb695